webhook/webhook.go

```diff
@@ -119,6 +123,16 @@ func New(
 		opts.StatsReporter = reporter
 	}
 
+	webhookTLSMinVersion := uint16(tls.VersionTLS13)
+	switch opts.TLSMinVersion {
+	case 0:
+		// No-op, the default value is already set
+	case tls.VersionTLS13, tls.VersionTLS12:
+		webhookTLSMinVersion = opts.TLSMinVersion
+	default:
+		return nil, fmt.Errorf("unsupported TLS version: %d", opts.TLSMinVersion)
+	}
+
```

Comment:
probably simpler to do something like

```go
if opt.TLSMinVersion == 0 {
    opts.TLSMinVersion = TLSMinVersionFromEnv(defaultTLSMinVersion)
}
```

Reply:
My concern was that someone would configure the minimum version to VersionTLS11, for example. Do you think this is a problem or there is no need to verify "unsupported versions"?

Reply:
yeah good point